src/org/exist/collections/CollectionCache.java

```diff
@@ -38,13 +39,13 @@
  * 
  * @author wolf
  */
-public class CollectionCache extends LRUCache {
+public class CollectionCache extends LRUCache implements BrokerPoolService {
 
     private Object2LongHashMap names;
     private BrokerPool pool;
 
     public CollectionCache(BrokerPool pool, int blockBuffers, double growthThreshold) {
-        super(blockBuffers, 2.0, 0.000001, CacheManager.DATA_CACHE);
+        super(blockBuffers, 2.0, growthThreshold, CacheManager.DATA_CACHE);
```

Comment:
my question is simple: how well you stress test db with that new parameter?

If you did not stress test db then change line 840 at `BrokerPool` to keep old parameter.